### ISSUE TITLE
One profile won't download

### ISSUE THREAD

**Issue by Asylum234** [HuggableSquare/vsco-dl#1]
Love your tool, it's worked for every single account I've tried except one. When I run it, regardless of switches, I get this error: 

PS C:\Users\Main\Desktop\vsco-dl-master\vsco-dl-master> .\vsco-dl.rb -m michdeveas
C:/Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:363:in `open_http': 408 Request Timeout (OpenURI::HTTPError)
        from C:/Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:741:in `buffer_open'
        from C:/Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:212:in `block in open_loop'
        from C:/Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:210:in `catch'
        from C:/Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:210:in `open_loop'
        from C:/Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:151:in `open_uri'
        from C:/Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:721:in `open'
        from C:/Ruby24-x64/lib/ruby/2.4.0/open-uri.rb:35:in `open'
        from C:/Users/Main/Desktop/vsco-dl-master/vsco-dl-master/vsco-dl.rb:36:in `<main>'

Immediately after this, I ran this:

PS C:\Users\Main\Desktop\vsco-dl-master\vsco-dl-master> .\vsco-dl.rb -m ahappycloud
Image 17 of 858

And it ran just fine. And the other 15-20 accounts had no problems either.

**Comment by HuggableSquare** [comment]
I did a little digging, and it seems that VSCO's servers were choking on requests for very large amounts of images (the account you mentioned has 2018 images total). I'm not sure the exact amount where it times out but it seems to be somewhere north of 1000, I managed to get a request for 1500 through at one point, but only after warming the cache. I modified the script to now ask for 1000 images, and then ask for the next 1000, and the next 1000, and so on until it has them all (instead of all of them at once like it did before).

Long story short, it now should work fine for accounts with lots of pictures. Let me know if you have any more issues with it though, this fix _should_ be relatively stable but that could change.

EDIT: Also apparently as an interesting side effect, it now includes the first image a person ever posted whereas I guess it didn't used to before. Hooray for fixing bugs I didn't realized existed! 🎉

**Comment by Asylum234** [comment]
Worked perfectly. Thank you so much!
